x-pack/plugins/watcher/public/application/sections/watch_status_page/components/execution_history_panel.tsx

```diff
@@ -135,18 +138,79 @@ export const ExecutionHistoryPanel = () => {
     },
     {
       field: 'watchStatus.state',
-      name: i18n.translate('xpack.watcher.sections.watchHistory.watchTable.stateHeader', {
-        defaultMessage: 'State',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.watcher.sections.watchHistory.watchTable.stateHeader.tooltipText',
+            {
+              defaultMessage: 'Active or error state.',
+            }
+          )}
+        >
+          <span>
+            {i18n.translate('xpack.watcher.sections.watchHistory.watchTable.stateHeader', {
+              defaultMessage: 'State',
+            })}{' '}
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      sortable: true,
+      truncateText: true,
+      render: (state: string) => <WatchStateBadge state={state} />,
+    },
+    {
+      field: 'startTime',
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.watcher.sections.watchHistory.watchTable.metConditionHeader.tooltipText',
+            {
+              defaultMessage: 'Whether the condition was met and action taken.',
+            }
+          )}
+        >
+          <span>
+            {i18n.translate('xpack.watcher.sections.watchHistory.watchTable.metConditionHeader', {
+              defaultMessage: 'Condition met',
+            })}{' '}
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
       sortable: true,
       truncateText: true,
-      render: (state: string) => <WatchStatus status={state} />,
+      // TODO: Once we convert the client-side models to TS, this should be a WatchHistoryItemModel.
+      render: (startTime: Moment, item: any) => {
+        const {
+          watchStatus: { lastExecution },
+        } = item;
+
+        if (startTime.isSame(lastExecution)) {
+          return <EuiIcon color="green" type="check" />;
+        }
+      },
     },
     {
       field: 'watchStatus.comment',
-      name: i18n.translate('xpack.watcher.sections.watchHistory.watchTable.commentHeader', {
-        defaultMessage: 'Comment',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.watcher.sections.watchHistory.watchTable.commentHeader.tooltipText',
+            {
+              defaultMessage:
+                'Whether the action was throttled, acknowledged, acknowledgeable, or failed to execute.',
```

Comment:
Is this the right order, or should it go from positive to negative:  acknowledged, acknowledgeable, throttled, or failed?  Should actions be singular or plural as the comment above?

> Whether the action was throttled, acknowledged, acknowledgeable, or failed.',

Reply:
Can we come up with another word for acknowledgeable?

Reply:
For context, some actions can be "Acknowledged". I made up the term "Acknowledgeable" to describe this. Another term known to our users is "Ackable".

I have no preference about the order, and I don't have a feeling that the order will affect the meaning extracted by the reader.

What do you mean by "Should actions be singular or plural as the comment above"? I can't find the referenced comment.